SYCL/ESIMD/api/functional/ctors/ctor_default.cpp

```diff
@@ -0,0 +1,156 @@
+//==------- ctor_default.cpp  - DPC++ ESIMD on-device test ----------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu, level_zero
```

Comment:
I think the syntax should be like this: https://github.com/intel/llvm-test-suite/blob/intel/SYCL/Regression/host_unified_memory.cpp#L1
 but if the comma works too, then fine with me:
```suggestion
// REQUIRES: gpu && level_zero
```
or even better this:
```suggestion
// REQUIRES: gpu
```
I don't see anything special in this test that would prevent it from running with opencl. Majority of ESIMD tests can run on both level_zero and opencl, e.g.: https://github.com/intel/llvm-test-suite/blob/intel/SYCL/ESIMD/vadd_1d.cpp#L8

Reply:
Due to OPenCL is not supported on simd constructor tests we can't use this requirements, but was added these strings:
```
// XREQUIRES: gpu
// TODO gpu and level_zero in REQUIRES due to only this platforms supported yet.
// The current "REQUIRES" should be replaced with "gpu" only as mentioned in
// "XREQUIRES".
```

they can be found in the commit: 84aa45dfae2fabf7818cc522be80a357d061e513